grails-app/migrations/changesets/changelog-11.0.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="vvicario" id="changelog-11.0-1">
+        <addColumn tableName="consent_collection_link">
+            <column name="pii" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="compliance" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="sharing_type" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="text_sharing_type" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="text_compliance" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="international_cohorts" type="LONGTEXT"/>
+        </addColumn>
+        <addColumn tableName="consent_collection_link">
+            <column name="require_mta" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="vvicario" id="changelog-11.0-2">
+        <addColumn tableName="storage_document">
+            <column name="consent_key" type="VARCHAR(255)"/>
+        </addColumn>
+    </changeSet>
```

Comment:
Don't know how I missed this file in the review, but this goes along with previous commentary - we don't need this new column. Consent key is stored in `issue_key`. Remember that Consent Groups are just `Issue` objects.